e2e/src/utils/retries.js

```diff
@@ -20,7 +22,7 @@ export const launchApp = async (
     },
     { retries: 5, delay: 10 * 1000, timeout: 30 * 10000 }
   ).then(async () => {
-    await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-alfajores.*'])
+    await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-mainnet.*'])
```

Comment:
what's the purpose of this? With the move to zerion and cloud functions, we'll likely not be hitting this url any longer right?

Reply:
Removed in d77be26d113fd7451939d4bd046c8e119701e397.